engine/test/integration_tests/dm_case_sensitive/run.sh

```diff
@@ -42,8 +43,6 @@ function run() {
 	exec_with_retry 'run_sql --port 4000 "select count(*) from UPPER_DB_ROUTE.do_table_route\G" | grep -Fq "count(*): 2"'
 }
 
-trap "stop_engine_cluster $CONFIG" EXIT
-run $*
-# TODO: handle log properly
-# check_logs $WORK_DIR
+trap "stop_engine_cluster $WORK_DIR $CONFIG" EXIT
+# run $*
```

Comment:
Why is this line comment, is this case still not available?

Reply:
will be enable again in https://github.com/pingcap/tiflow/pull/6857